public/components/ResultsComparison.vue

```diff
@@ -8,7 +8,8 @@
       :variables="variables"
       class="view-toggle"
     >
-      <p class="font-weight-bold mr-auto">Samples</p>
+      <p class="font-weight-bold">Samples</p>
+      <legend-shap v-if="isTableView" class="ml-5 mr-auto" />
```

Comment:
I don't know if this `isTableView` is necessary. I don't know if we are displaying SHAP on result other than TABLE_VIEW.